src/web.md

```diff
@@ -1,82 +1,104 @@
 ---
-title: Flutter for web
+title: Adding web support to a Flutter app
 description: "Announcing the preview release of Flutter for web."
 ---
 
-Flutter for web is a code-compatible implementation of Flutter that is
+Web support is a code-compatible implementation of Flutter that is
 rendered using standards-based web technologies: HTML, CSS and JavaScript.
-With Flutter for web, you can compile existing Flutter code written in
+With web support, you can compile existing Flutter code written in
 Dart into a client experience that can be embedded in the browser and
 deployed to any web server. You can use all the features of Flutter,
 and you don’t need a browser plug-in.
 
 {{site.alert.warning}}
-  Flutter for web is currently available as a technical preview.
-  When trying Flutter for web, you can expect to experience crashes
-  and missing features. For more information, see the [Flutter for
-  web README]({{site.github}}/flutter/flutter_web/blob/master/README.md).
+  **As of 1.9, web support is currently available as early release.**
```

Comment:
throughout the PR we want to call it "early tech preview", not "release" or "support".